web/ui/react-app/src/components/SearchBar.tsx

```diff
@@ -1,29 +1,34 @@
-import React, { ChangeEvent, FC } from 'react';
+import React, { ChangeEvent, FC, useEffect } from 'react';
 import { Input, InputGroup, InputGroupAddon, InputGroupText } from 'reactstrap';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faSearch } from '@fortawesome/free-solid-svg-icons';
 
 export interface SearchBarProps {
-  handleChange: (e: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => void;
+  handleChange: (e: string) => void;
   placeholder: string;
+  defaultValue: string;
 }
 
-const SearchBar: FC<SearchBarProps> = ({ handleChange, placeholder }) => {
+const SearchBar: FC<SearchBarProps> = ({ handleChange, placeholder, defaultValue }) => {
   let filterTimeout: NodeJS.Timeout;
 
   const handleSearchChange = (e: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => {
     clearTimeout(filterTimeout);
     filterTimeout = setTimeout(() => {
-      handleChange(e);
+      handleChange(e.target.value);
     }, 300);
   };
 
+  useEffect(() => {
+    handleChange(defaultValue);
+  }, []);
```

Comment:
Actually now that I see this we should probably include `defaultValue` in the deps here to be really correct in case the `defaultValue` gets updated from the outside, although I think it can never happen since we also memo-ize the default value in the parent :grimacing: 

```suggestion
  }, [defaultValue]);
```

Reply:
sorry, don't sign and damage your suggestion, I'll upload it on my behalf sorry, thanks for the correction